content/en/docs/concepts/storage/volumes.md

```diff
@@ -1143,9 +1149,9 @@ persistent volume:
   The value is passed as `volume_id` on all calls to the CSI volume driver when
   referencing the volume.
 * `readOnly`: An optional boolean value indicating whether the volume is to be
-  "ControllerPublished" (attached) as read only. Default is false. This value is
-  passed to the CSI driver via the `readonly` field in the
-  `ControllerPublishVolumeRequest`.
+  "ControllerPublished" (attached) as read only. Default is false (with the
+  exception of the `configMap` and `secret` volume types). This value is passed
```

Comment:
I think this section is explicitly talking about CSI volume type; it might not be appropriate to include this detail here

Reply:
I agree (great first attempt eh?)

I'll commit this up removed 👍